docs/DEV-3.-Frontend.md

```diff
@@ -1 +1,111 @@
 # Frontend Deployment for Development
+
+
+## The Microfrontend configuration
+The Field Mapping Tasking Manager is a web application that helps users coordinate mapping efforts in OpenStreetMap. It is built using a microfrontend architecture, which means that the application is composed of multiple small frontend modules that can be developed, tested, and deployed independently.
+
+FMTM uses Webpack remote modules to dynamically load code from other microfrontend applications, which allows developers to develop, test, and deploy smaller and more manageable parts of code independently. This improves the performance and scalability of the application.
```

Comment:
Thanks for the review! 

Should we keep the explanation of microfrontend configuration? Also, I'm unsure of the list of current modules, could you please walk me through them? Thanks again :)

List of current microfrontend modules:

**main:**
Description: The main frontend, displaying projects and tasks.
Port: 8080.

**fmtm_openlayers_map:**
Description: The map component, displaying tasks on a map.
Port: 8081.

**fmtm_header:**
Description: The header component, providing navigation and user profile options.
Port: 8082.

fmtm_project_list:
Description: The project list component, displaying a list of available projects.
Port: 8083.

Reply:
We can keep the explanation - thanks! It can always be refined with more info later.

I think only the two microfrontends I described are present: https://github.com/hotosm/fmtm/tree/main/src/frontend
I can't see `fmtm_header` or `fmtm_project_list` you mentioned.